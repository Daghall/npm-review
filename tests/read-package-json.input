{
  "dependencies": {
    "express": "^4.17.1",
    "uuid": "npm:uuid/v4@0.1.0"
  },
  "devDependencies": {
    "mocha": "~9.0.0",
    "nodemon": "^1.2.3"
  }
}
