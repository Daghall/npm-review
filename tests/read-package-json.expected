express	4.17.1	.dependencies
uuid	0.1.0	.dependencies	uuid/v4
mocha	9.0.0	.devDependencies
nodemon	1.2.3	.devDependencies
